support-frontend/assets/pages/digital-subscription-checkout/components/digitalCheckoutForm.test.tsx

```diff
@@ -79,6 +90,8 @@ describe('Digital checkout form', () => {
 			client_secret: 'super secret',
 		});
 
+		mock(isSwitchOn).mockImplementation(() => true);
```

Comment:
I don't suppose there's any chance of testing some of the different potential switch states? I know this has been a bit of a pain to get behaving so no worries if you think it'd be too complicated.

Reply:
This has been a bit of a pain to make happy but I agree it would be great to explicitly test different potential switch states! I'm happy to get in a card and do a follow up PR